content/en/blog/_posts/2023-08-15-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,295 @@
+* [Support 3rd party device monitoring plugins](https://github.com/kubernetes/enhancements/issues/606)
+* [Auth API to get self-user attributes](https://github.com/kubernetes/enhancements/issues/3325)
+* [Proxy Terminating Endpoints](https://github.com/kubernetes/enhancements/issues/1669)
+* [Expanded DNS Configuration](https://github.com/kubernetes/enhancements/issues/2595)
+* [Cleaning up IPTables Chain Ownership](https://github.com/kubernetes/enhancements/issues/3178)
+* [Minimizing iptables-restore input size](https://github.com/kubernetes/enhancements/issues/3453)
+* [Graduate the kubelet pod resources endpoint to GA](https://github.com/kubernetes/enhancements/issues/3743)
+* [Extend podresources API to report allocatable resources](https://github.com/kubernetes/enhancements/issues/2403)
+* [Move EndpointSlice Reconciler into Staging](https://github.com/kubernetes/enhancements/issues/3685)
+
+### Deprecations and removals
+
+Removals:
+
+* [Removal of CSI Migration for GCE PD](https://github.com/kubernetes/enhancements/issues/1488)
+
+Deprecations:
+
+* [Ceph RBD in-tree plugin](https://github.com/kubernetes/kubernetes/pull/118303)
+* [Ceph FS in-tree plugin](https://github.com/kubernetes/kubernetes/pull/118143)
+
+## Release Notes
+
+The complete details of the Kubernetes v1.28 release are available in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md).
+
+## Availability
+
+Kubernetes v1.28 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.28.0). To get started with Kubernetes, you can run local Kubernetes clusters using [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), etc. You can also easily install v1.28 using [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+ 
+## Release Team
+
+Kubernetes is only possible with the support, commitment, and hard work of its community. Each release team is comprised of dedicated community volunteers who work together to build the many pieces that make up the Kubernetes releases you rely on. This requires the specialized skills of people from all corners of our community, from the code itself to its documentation and project management.
+
+We would like to thank the entire release team for the hours spent hard at work to ensure we deliver a solid Kubernetes v1.28 release for our community.
+
+Special thanks to our release lead, Grace Nguyen, for guiding us through a smooth and successful release cycle and facilitating the communication and collaboration between the different release teams taking advantage of her experience as a shadow and lead in these teams. If you would like to learn more about Grace's journey and this release, listen to the [Kubernetes v1.28 release interview podcast](https://craigbox.substack.com/p/the-kubernetes-128-release-interview).
```

Comment:
Hi folks, I actually would prefer to not include the podcast in the blog since it's a third-party source. I'll did multiple interviews for this release so would like to remain fair to all of them.

cc @craigbox @sftim @bradmccoydev

Reply:
I'm fine with either:
- we link to each interview
- we omit the interview from the announcement

Reply:
I agree with @gracenng, I don't think it's appropriate to link to a single interview in this release blog and would like to see this dropped.

Reply:
Please remove this from the PR.